content/admin/configuration/configuring-your-enterprise/configuring-rate-limits.md

```diff
@@ -61,3 +63,29 @@ You can apply Git rate limits per repository network or per user ID. Git rate li
 3. Type limits for each repository network or user ID.
   ![Fields for repository network and user ID limits](/assets/images/enterprise/management-console/example-git-rate-limits.png)
 {% data reusables.enterprise_management_console.save-settings %}
+
+## Enabling Actions rate limits
+
+You can apply a rate limit to GitHub Actions workflow runs. The Actions rate limit is expressed in runs per minute. It is for the sum total of all Actions runs across the entire GHES instance.
+
+By default, the rate limit is disabled. When workflow jobs cannot immediately be assigned to an available runner, they will wait in a queue until a runner is available. However, if Actions experiences a sustained high load, the queue can back up faster than it can drain and the performance of the GHES instance may degrade.
+
+To avoid this, an administrator can configure a rate limit. From the [administrative shell (SSH)](/enterprise/{{ currentVersion }}/admin/guides/installation/accessing-the-administrative-shell-ssh/), run the following commands:
+
+```
+actions-rate-limiting.enabled true
+actions-rate-limiting.queue-runs-per-minute 200
+ghe-config-apply
+```
+
+ When the rate limit is exceeded, additional workflow runs will fail immediately rather than being put in the queue:
+
+![screenshot of a rate-limited run](/assets/images/enterprise/actions/actions-rate-limited.png)
```

Comment:
I took off the `$` in the first block because the other ones don't have it.  I also don't like when code snippets have that because it messes up copy-paste.  But happy to go with whatever our convention is.

> I think the `ghe-config` command is missing for the first two configuration values

I'm pretty sure I didn't use that when I tested it because I just copied the same snippet here 🤔
Now I don't know why that should work.  Having `ghe-config` shouldn't hurt, but I can double check.